spec/features/user_sign_in_feature_spec.rb

```diff
@@ -1,9 +1,16 @@
 require 'spec_helper'
 
-feature "user signs in" do 
-  scenario "with valid email and password" do 
+feature "user signs in" do
+  scenario "with valid email and password" do
     alice = Fabricate(:user)
     sign_in(alice)
-    page.should have_content alice.full_name 
+    page.should have_content alice.full_name
```

Comment:
You should be using `expect` syntax: `expect(page).to have_content alice.full_name`.